### PR TITLE
fix 'net' uninitialized on kernel<4.4

### PR DESCRIPTION
**Pull request by llccd**

### CODE REVIEW

**Comment by LGA1150** [comment]
`net` is not used on kernel < 4.4. Perhaps I should remove it?

**Comment by llccd** [comment]
`net` is used when MASQUERADE workaround enabled on IPv6. If `net` is uninitialized, kernel<4.4 will crash when enable MASQUERADE workaround on IPv6.

File: libxt_SPOOFTCP.c [hunk 1]
```diff
@@ -31,6 +31,8 @@ static const struct tcp_flag_names tcp_flag_names[]
     { "PSH", 0x08 },
     { "ACK", 0x10 },
     { "URG", 0x20 },
+    { "ECE", 0x40 },
+    { "CWR", 0x80 },
     { "ALL", 0x3F },
```
Comment (LGA1150) [inline libxt_SPOOFTCP.c]: Should be 0xFF if the additional 2 flags are added

File: libxt_SPOOFTCP.c [hunk 2]
```diff
@@ -31,6 +31,8 @@ static const struct tcp_flag_names tcp_flag_names[]
...
     { "NONE", 0 },
 };
```
Comment (LGA1150) [inline libxt_SPOOFTCP.c]: Set `TCP_FLAG_NAMES_SIZE` to 10

**Comment by LGA1150** [comment]
Please squash these 2 commits "Add ECE,CWR flags", "Change TCP_FLAG_NAMES_SIZE to 10" into one

**Pull request merged**